src/EnergyPlus/ChillerReformulatedEIR.cc

```diff
@@ -342,56 +355,77 @@ namespace ChillerReformulatedEIR {
 				ErrorsFound = true;
 			}
 
-			ElecReformEIRChiller( EIRChillerNum ).EIRFPLRName = cAlphaArgs( 4 );
-			ElecReformEIRChiller( EIRChillerNum ).ChillerEIRFPLR = GetCurveIndex( cAlphaArgs( 4 ) );
+			//The default type of part-load curve is: LeavingCondenserWaterTemperature
+			if ( lAlphaFieldBlanks( 4 ) ) {
+				PartLoadCurveType = "LeavingCondenserWaterTemperature";
+			}
+			else {
+				PartLoadCurveType = cAlphaArgs( 4 );
+			}
+			
+			ElecReformEIRChiller( EIRChillerNum ).EIRFPLRName = cAlphaArgs( 5 );
+			ElecReformEIRChiller( EIRChillerNum ).ChillerEIRFPLR = GetCurveIndex( cAlphaArgs( 5 ) );
 			if ( ElecReformEIRChiller( EIRChillerNum ).ChillerEIRFPLR == 0 ) {
 				ShowSevereError( RoutineName + cCurrentModuleObject + "=\"" + cAlphaArgs( 1 ) + "\"" );
-				ShowContinueError( "Invalid " + cAlphaFieldNames( 4 ) + '=' + cAlphaArgs( 4 ) );
+				ShowContinueError( "Invalid " + cAlphaFieldNames( 5 ) + '=' + cAlphaArgs( 5 ) );
 				ErrorsFound = true;
 			}
-
-			// Chilled water inlet/outlet node names are necessary
-			if ( lAlphaFieldBlanks( 5 ) ) {
+			
+			//Check the type of part-load curves implemented: 1_LeavingCondenserWaterTemperature, 2_Lift    zrp_Aug2014
+			if ( SameString( PartLoadCurveType, "LeavingCondenserWaterTemperature" ) && SameString( GetCurveType( ElecReformEIRChiller( EIRChillerNum ).ChillerEIRFPLR ), "BICUBIC" ) ) {
```

Comment:
@rongpengzhang Didn't notice these before.  These if blocks need to be expanded to include the valid table types.  Some components allow any form a curve as long as it has the same number of independent variables.  Do you feel strongly that this should only allow these specific curve forms?  e.g.  Should biquadratic be illegal, or just a warning, or completely acceptable?  @mmatts @Myoldmopar @EnergyArchmage Comments?